<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceAllPortsPage</class>
 <widget class="QWidget" name="MaintainceAllPortsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
		background-color:rgb(0, 0, 50);
}
QPushButton
{
font: 20px, &quot;微软雅黑&quot;;
color: white;
background-color: rgb(0, 128, 128);
border-top: 2px solid white;
border-left: 2px solid white;
border-radius:8px;
}
</string>
  </property>
  <widget class="QPushButton" name="BTN_HMI">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>HMI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_ERM">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>270</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>ERM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_BCU">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>420</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>BCU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Return">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>703</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_RIOM">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>170</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>RIOM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>320</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DCU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_HVAC">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>620</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>HVAC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_ATC">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>ATC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CCUA">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>CCUA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_EDCUA">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>470</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>EDCUA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SIV">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>370</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>SIV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_PIS">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>710</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>PIS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CCU_D">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>570</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>CCU-D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CCUB">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>70</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>CCUB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_TDS">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>710</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TDS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_LCUA">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>710</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>LCUA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_BMS">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>710</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>BMS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_FAS">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>710</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>FAS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_PCU">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>710</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>PCU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_EDCUB">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>520</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>EDCUB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_LCUB">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>710</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>LCUB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CCUC">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>120</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>CCUC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
